packages/ui/package.json

```diff
@@ -10,7 +10,9 @@
   ],
   "main": "app/main.js",
   "scripts": {
-    "postinstall": "npm link ../daemon",
+    "postinstall": "run-s electron-rebuild link-daemon",
+    "link-daemon": "npm link --force ../daemon",
```

Comment:
I don't like using `--force` but `npm link` would error out with the following:

```
npm ERR! code EEXIST
npm ERR! path /home/gus/repos/awala-gateway-desktop/packages/daemon/node_modules/.bin/pino
npm ERR! Refusing to delete /home/gus/repos/awala-gateway-desktop/packages/daemon/node_modules/.bin/pino: containing path /home/gus/repos/awala-gateway-desktop/packages/daemon/node_modules/pino isn't under npm's control
npm ERR! File exists: /home/gus/repos/awala-gateway-desktop/packages/daemon/node_modules/.bin/pino
npm ERR! Remove the existing file and try again, or run npm
npm ERR! with --force to overwrite files recklessly.
```

But even after passing `--force`, I still had to delete `packages/daemon/node_modules` by hand to make `npm install` work.